### PR TITLE
Kkaune/getting started

### PR DESCRIPTION
**Pull request by kkraune**

### CODE REVIEW

**Review by hmusum — APPROVED** [review]

File: en/vespa-quick-start.html [hunk 1]
```diff
@@ -126,9 +125,9 @@ <h2 id="running-vespa-in-docker">Running Vespa in Docker</h2>
   <div class="pre-parent">
     <button class="d-icon d-duplicate pre-copy-button" onclick="copyPreContent(this)"></button>
 <pre data-test="exec">
-$ vespa document src/test/resources/A-Head-Full-of-Dreams.json
-$ vespa document src/test/resources/Love-Is-Here-To-Stay.json
-$ vespa document src/test/resources/Hardwired...To-Self-Destruct.json
+$ vespa document src/test/resources/A-Head-Full-of-Dreams.json &amp;&amp; \
```
Comment (bratseth) [inline en/vespa-quick-start.html:128]: I don't agree this is to simplify - it's more syntax and harder to understand.

File: en/vespa-quick-start.html [hunk 2]
```diff
@@ -104,8 +104,7 @@ <h2 id="running-vespa-in-docker">Running Vespa in Docker</h2>
   <div class="pre-parent">
     <button class="d-icon d-duplicate pre-copy-button" onclick="copyPreContent(this)"></button>
 <pre data-test="exec">
-$ vespa clone album-recommendation myapp
-$ cd myapp
+$ vespa clone album-recommendation myapp &amp;&amp; cd myapp
```
Comment (bratseth) [inline en/vespa-quick-start.html:107]: I wouldn't call this to simplify either - just slightly harder to understand.

Comment (kkraune) [reply en/vespa-quick-start.html:107]: ditto

**Comment by bratseth** [comment]
Well we should change the framework then.

**Pull request merged**